gps/source_manager.go

```diff
@@ -216,7 +218,7 @@ func (sm *SourceMgr) HandleSignals(sigch chan os.Signal) {
 
 				opc := sm.suprvsr.count()
 				if opc > 0 {
-					fmt.Printf("Signal received: waiting for %v ops to complete...\n", opc)
+					sm.logf("Signal received: waiting for %v ops to complete...\n", opc)
```

Comment:
sorry, this one has to stay as-is. yes, it's obnoxious to print directly in this way, but the interface here is carefully designed; we're not injecting a logger for this one, narrow case. if it's that bad, we'll just delete this logging output entirely.